examples/joy-next-app-router-ts/src/app/layout.tsx

```diff
@@ -11,7 +11,7 @@ export const metadata = {
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
   return (
-    <html lang="en">
+    <html lang="en" className={publicSans.variable}>
       <body className={publicSans.className}>
```

Comment:
Shouldn't we remove `publicSans.className`?

Reply:
I think it might be a good idea to leave it as it is, firstly to show in an example how to use/configure "next/font", and secondly, if someone wants to add more text content inside the body, they will see that the font remains the same.